CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Bugfixes
+
+- [#1463](https://github.com/influxdata/telegraf/issues/1463): Shared WaitGroup in Exec plugin
```

Comment:
@sparrc 
> this should go in the 1.0 bugfixes section

Why? this "bug" only happens if you run a modified version of telegraf.